### DISCUSSION TITLE
APOE: regression

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1501] [General]
"Pipeline error from pipeline 6335da001b282a216c65b81d user: krootesmurdy\n Error details: Computation failed with exitcode 1 and stderr Traceback (most recent call last):\n File \"/computation/scripts/local.py\", line 189, in <module>\n sys.stdout.write(local_1(PARAM_DICT))\n File \"/computation/scripts/local.py\", line 74, in local_1\n _, local_stats_list = local_stats_to_dict_numba(args, encoded_X, y)\n File \"/computation/scripts/local_ancillary.py\", line 94, in local_stats_to_dict_numba\n params, _, tvalues, rsquared, dof_global = gather_local_stats(X1, y)\n File \"/usr/local/lib/python3.7/site-packages/numba/np/linalg.py\", line 574, in dot_3_mv_check_args\n raise ValueError(\"incompatible array sizes for np.dot(a, b) \"\nValueError: incompatible array sizes for np.dot(a, b) (matrix * vector)\n"

unclear as to what this is related to.

**Comment by hvgazula** [comment]
Related to regression. The sizes of X and y are not the same.

**Comment by rssk** [comment]
I think these are typically when the files input and the csv disagree on subj count or perhaps other things

**Reply by krootesmurdy** [reply]
but shouldn't the files that are inputted be read in from the csv file? 

maybe the mismatch was from the missing cells? some participants didn't have all their demographic data so some of the covariates length did not match.
